core/jazz_t-vault/swagger/swagger.json

```diff
@@ -0,0 +1,3589 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2017-01-30T23:30:18Z",
+        "title": "{api_deployment_node_title}",
+        "description": "API specification for jazz t-valut api"
+    },
+    "host": "{api_host_name}",
+    "basePath": "/{domain}",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/{service_name}/safes": {
+            "options": {
+                "description": "CORS Support",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
```

Comment:
Let's use the following value for tags for each API/method - 
"tags" : ["Secret Management"]

Reply:
Done